iib/web/api_v1.py

```diff
@@ -267,6 +288,9 @@ def add_bundles():
     if not isinstance(payload, dict):
         raise ValidationError('The input data must be a JSON object')
 
+    if 'bundles' in payload:
+        payload['bundles'] = _get_unique_bundles(payload['bundles'])
```

Comment:
Why not do this in `from_json`?

Reply:
Yeah, it might be better place.

Reply:
I take it back, for example here in `add_bundles` there are also other methods/functions which takes payload as an input parameter. And if I remove duplicates just for `RequestAdd.from_json()` then it would not be changed for the rest. Therefore my suggestion is to keep it at the beginning and remove duplicates from payload before any processing.

Reply:
The reason I say this is it would be much easier to handle in `RequestAdd.from_json()` since all you have to do is probably cast it into a set and then convert it to a list again. I think OPM is capable of handling bundles out of order. And then you can use the bundles from the returned request. Or you can simply do a `list(set(payload['bundles']))` here. wdyt?

Reply:
what I was referring to was that when I remove duplicates just in the `RequestAdd.from_json()` then it will not be removed from payload (at least this behavior is not expected). 
And if we do it this way then we will call ` handle_add_request.apply_async(args=args, ...` with different data (including duplicated bundles) which might not be good idea.
Therefore I chose this approach removing it in payload at the beginning of request.